src/lipalightninglib.udl

```diff
@@ -507,6 +511,7 @@ enum RuntimeErrorCode {
     // CONNECTIVITY ERRORS
     "AuthServiceUnavailable", // The backend auth service is unavailable.
     "TopupServiceUnavailable", // The topup service is unavailable.
+    "NotificationServiceUnavailable", // The notification service is unavailable.
```

Comment:
I realize that other error codes are not better but anyway: What kind of additional information does the comment provide?